app/contrib/ds/admin_style/templates/admin/filer/widgets/admin_file.html

```diff
@@ -20,7 +20,7 @@
          {% if max_filesize %}}data-max-filesize="{{ max_filesize|safe }}"{% endif %}>
         <div class="z-index-fix"></div>
         <div class="dz-default dz-message js-filer-dropzone-message{% if object %} hidden{% endif %}">
-            <span class="icon filer-icon filer-icon-arrow-down fa fa-arrow-down"></span><span>{% translate "or drop your file here" %}</span>
+            <span class="icon filer-icon filer-icon-arrow-down fa fa-upload"></span><span> Selecione um arquivo ou arraste-o para cá</span>
```

Comment:
talvez vale colocar um espaço entre os spans para considerar o espaçamento entre icon e texto. e também colocar o `translate`!
```suggestion
            <span class="icon filer-icon filer-icon-arrow-down fa fa-upload"></span> <span>{% translate "Select or drop your file here" %}</span>
```